Add information about constraint usage to install older napari release (#193)
napari/docs: docs/tutorials/fundamentals/installation.md

*(See [Choosing a different Qt backend](#choosing-a-different-qt-backend) below for an explanation of the `[all]`
notation.)*

> *(See[Using constraints file](#using-constraints-file) for trubleshot with install older version of napari)*

Reviewer: Feel free to ignore if you mean something different.

```suggestion
*(See[Using constraints file](#using-constraints-file) for more information on installing older versions of napari)*
```

Author: I try to told that this information is most important for people who have a problem with setup environment. This is why I use "troubleshooting". But I'm not sure if it is required to highlight this scenario.

Reviewer: Oh, feel free to ignore then :) Thanks again!

you're not using.
```

> ## Using constraints file
> 
> Since napari 0.4.18 we are storing constraints file with information against which exact depenedcy version napari was tested. 
> It could be usefull if you need to install napari as package from pypi and ends with environment that napari does not start or work properly. 
> 
> The constraints files are stored in repository `resources/constraints/constraints_py3.10.txt`. To find
> constraints for interesting relese go under the link `https://github.com/napari/napari/tree/{tag}/resources/constraints` 
> With substitute `{tag}` with looked napari version. 
> 
> ```sh
> pip install napari[backend_selection] -c path/to/constraints/file
> ```
> 
> For example, when you would like to install napari on python 3.10:

Reviewer: ````suggestion
## Using constraints files

Since napari 0.4.18, we store constraints files with information about each exact dependency version against which napari was tested.
This could be useful if you need to install napari as a package from PyPI, and prevents creating environments where napari does not start or work properly.

The constraints files are stored in the napari repository under `resources/constraints/constraints_py3.10.txt`. To find
constraints for specific releases, go under the link `https://github.com/napari/napari/tree/{tag}/resources/constraints`
replacing `{tag}` with the desired napari version.

```sh
pip install napari[backend_selection] -c path/to/constraints/file
```

For example, if you would like to install napari on python 3.10:
````
